requirements.txt

```diff
@@ -1,3 +1,5 @@
+pip freeze > requirements.txt
+
```

Comment:
When did you run pip freeze? I thought there might be more modules in this file but maybe not.